docs-next/components/Navigation.tsx

```diff
@@ -1,39 +1,61 @@
-import React, { Fragment, ReactNode } from 'react';
-import Link from 'next/link';
+/** @jsx jsx */
+import { Fragment, ReactNode } from 'react';
 import { useRouter } from 'next/router';
-import cx from 'classnames';
+import { jsx } from '@keystone-ui/core';
+import Link from 'next/link';
 
 type SectionProps = { label: string; children: ReactNode };
-const Section = ({ label, children }: SectionProps) => {
+function Section({ label, children }: SectionProps) {
   return (
     <Fragment>
-      <h3 className="uppercase mt-6 mb-2 text-gray-700 font-bold">{label}</h3>
+      <h3
+        css={{
+          textTransform: 'uppercase',
+          margin: '1.5rem 0 0.5rem 0',
```

Comment:
Not a blocker, but it feels like a step backwards replacing spacing tokens with hard coded values 🙁 

We'll take a step forward again in a future PR?

Reply:
Yeah was thinking the same thing.
I did add some spacing tokens later but not applied them consistently ...
Needs an overall system PR at some point with font sizes as well